napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1641,3 +1641,102 @@ def get_optics(self):
             optics_detail[port] = port_detail
 
         return optics_detail
+
+    def get_interfaces_counters(self):
+        """
+        Return interface counters and errors.
+
+        'tx_errors': int,
+        'rx_errors': int,
+        'tx_discards': int,
+        'rx_discards': int,
+        'tx_octets': int,
+        'rx_octets': int,
+        'tx_unicast_packets': int,
+        'rx_unicast_packets': int,
+        'tx_multicast_packets': int,
+        'rx_multicast_packets': int,
+        'tx_broadcast_packets': int,
+        'rx_broadcast_packets': int,
+        """
+        if_mapping = {
+            'eth': {
```

Comment:
I think Black doesn't like this, can you upgrade to the latest version then run `black .`

Reply:
@mirceaulinic  Thanks for the review.
I fix it asap.

Reply:
@mirceaulinic Fixes on their way with latest push.